packages/table/src/table.tsx

```diff
@@ -542,6 +542,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             this.bodyElement,
             this.grid,
         );
+        this.locator = locator;
 
         const viewportRect = locator.getViewportRect();
         this.setState({ locator, viewportRect });
```

Comment:
Because we after we first render the table the first time, we need to pass the locator down, as props, to the table body and headers, for use in interactions. At this point, we haven't re-written the code to be able to use a locator that just lives in Table proper. 

We can discuss this offline if it's still confusing?